Marlin/ubl_G29.cpp

```diff
@@ -1341,15 +1339,7 @@
           // Also for round beds, there are grid points outside the bed that nozzle can't reach.
           // Prune them from the list and ignore them till the next Phase (manual nozzle probing).
 
-//        if ((probe_as_reference && position_is_reachable_by_probe_raw_xy(mx, my)) || position_is_reachable_raw_xy(mx, my))
-//          continue;
-//
-//        THE ABOVE CODE IS NOT A REPLACEMENT FOR THE CODE BELOW!!!!!!!
-//
-          bool reachable = probe_as_reference ?
-                             position_is_reachable_by_probe_raw_xy( mx, my ) :
-                             position_is_reachable_raw_xy( mx, my );
-          if ( ! reachable )
+          if ( ! (probe_as_reference ? position_is_reachable_by_probe_raw_xy(mx, my) : position_is_reachable_raw_xy(mx, my)) )
```

Comment:
I can't find a setting in the arduino editor to auto-chop trailing whitespace like most every other editor.

I agree it would be nice to have it auto-chop similar to how it already converts windows style EOL <CR><LF> to unix style <LF>.